packages/cardano-services/src/Utxo/UtxoHttpService.ts

```diff
@@ -0,0 +1,52 @@
+import * as OpenApiValidator from 'express-openapi-validator';
+import { Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { DbSyncUtxoProvider } from './DbSyncUtxoProvider';
+import { HttpServer, HttpService } from '../Http';
+import { Logger, dummyLogger } from 'ts-log';
+import { ServiceNames } from '../Program';
+import { providerHandler } from '../util';
+import express from 'express';
+import path from 'path';
+
+export interface UtxoServiceDependencies {
+  logger?: Logger;
+  utxoProvider: DbSyncUtxoProvider;
+}
+
+export class UtxoHttpService extends HttpService {
+  #utxoProvider: DbSyncUtxoProvider;
+
+  private constructor({ utxoProvider, logger = dummyLogger }: UtxoServiceDependencies, router: express.Router) {
+    super(ServiceNames.Utxo, router, logger);
+    this.#utxoProvider = utxoProvider;
+  }
+
+  async healthCheck() {
+    return this.#utxoProvider.healthCheck();
+  }
+
+  static create({ logger = dummyLogger, utxoProvider }: UtxoServiceDependencies) {
+    const router = express.Router();
+    const apiSpec = path.join(__dirname, 'openApi.json');
+    router.use(
+      OpenApiValidator.middleware({
+        apiSpec,
+        ignoreUndocumented: true, // otherwhise /metrics endpoint should be included in spec
+        validateRequests: true,
+        validateResponses: true
+      })
+    );
+    router.post(
+      '/search',
```

Comment:
```suggestion
      '/utxoByAddresses',
```
The HTTP API is expressing the provider interface

Reply:
I would use another format for API endpoints name: `/utxo-by-addresses`. WDYT?

Reply:
Yes, sorry that's the correct formatting. I've edited the suggestion